server/pbench/bin/pbench-sync-package-tarballs

```diff
@@ -18,7 +18,7 @@ esac
 # TOP, TMP, LOGSDIR, ARCHIVE, TS are all defined in pbench-base.sh
 . $dir/pbench-base.sh
 
-remotearchive=$ARCHIVE
+remotearchive=$ARCHIVE/hostnames
```

Comment:
We might need new versions of the sync scripts: we leave the old ones alone, to be dealt by the old cron scripts. If we can actually make the new version of `pbench-sync-satellite` use the old version of `pbench-sync-package-tarballs`, we should certainly do so, but the old versions should *NOT* be modified: they are needed to deal with the still extant old version of pbench-agent which will continue to deposit results in the old structure.